packages/datagateway-common/src/handleICATError.test.ts

```diff
@@ -0,0 +1,92 @@
+import { AxiosError } from 'axios';
+import * as log from 'loglevel';
+import handleICATError from './handleICATError';
+import { AnyAction } from 'redux';
+
+jest.mock('loglevel');
+
+describe('handleICATError', () => {
+  let error: AxiosError;
+  let events: CustomEvent<AnyAction>[] = [];
+
+  beforeEach(() => {
+    events = [];
+
+    document.dispatchEvent = (e: Event) => {
+      events.push(e as CustomEvent<AnyAction>);
+      return true;
+    };
+
+    error = {
+      isAxiosError: true,
+      config: {},
+      response: {
+        data: {},
+        status: 500,
+        statusText: 'Internal Server Error',
+        headers: {},
+        config: {},
+      },
+      name: 'Test error name',
+      message: 'Test error message',
+    };
+  });
+
+  it('logs an error and sends a notification to SciGateway', () => {
+    handleICATError(error);
+
+    expect(log.error).toHaveBeenCalledWith('Test error message');
```

Comment:
This message has been defined for this test - essentially it's a mock error. In reality this will be passed a real axios error object which will contain more useful error information

Reply:
As @louise-davies notes-these are test error messages, not to send to the user. I will close this conversation.